src/app/actions/user.actions.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { AnyAction } from 'redux';
+import { IUser } from '../models/user.model';
+import { UserActions } from './user.actions';
+
+describe('User Actions', () => {
+    beforeEach(()=> {
+        this.userActions = new UserActions();
+    });
+
+    it('should give an LOGIN_USER_REQUEST action', () => {
+        const username: string = 'bob';
```

Comment:
Especially when dealing with primitives and assigning initial values, unless that type has the possibility to change, or we're expecting very specific string values, we can let typescript infer the type here - by assigning 'bob' typescript will know that `this.username` must be a string

Of course this can be an opinionated thing - I err on the side of, if typescript can do the heavy lifting for me, let it